remove-testing-images/action.yml

```diff
@@ -0,0 +1,15 @@
+name: "remove-testing-images"
+description: "Remove a testing image by tag or clean up all untagged images"
```

Comment:
The description can be misleading because this action also removes images that have a single tag (thus, not only "untagged").